services/app-web/src/pages/StateSubmission/StateSubmissionForm.tsx

```diff
@@ -271,7 +268,7 @@ export const StateSubmissionForm = (): React.ReactElement => {
         )
     }
 
-    if (updateError && !showPageErrorMessage) {
+    if ((updateError || updateFormDataError) && !showPageErrorMessage) {
         setShowPageErrorMessage(true)
```

Comment:
Can you console error different messages for these two cases similar to the SubmissionSummary page error handling? We want to know (at least from the console and ideally also in logs) when the error is an api request fail versus unwrapping the form data on the client

Reply:
I’m a little confused about what updateError and updateFormDataError are, actually.